demo/Makefile

```diff
@@ -74,7 +74,7 @@ token:
 
 # Open dashboard in browser
 dashboard:
-	@open http://dashboard.${CLUSTER_DOMAIN}?token=${$(MAKE) token}
+	@URL=http://dashboard.${CLUSTER_DOMAIN}?token=${$(MAKE) token} && xdg-open $$URL || open $$URL
```

Comment:
As far as I know, it's not possible to set a make variable based on executable bash conditions.  You can only set shell & environment variables in that context, but I'll try that...